docs/design/sustainability/README.md

```diff
@@ -1,75 +1,57 @@
+| | For systems migrating to a hyperscale cloud provider | Consider physically shipping data to the provider | [Networking](./sustainable-engineering-principles.md#networking) | [Azure Data Box](https://azure.microsoft.com/en-us/services/databox/) | [Understanding data shipping tradeoffs](./sustainable-action-disclaimers.md#action-consider-physically-shipping-data-to-the-provider) |
+| | For [time-flexible workloads](https://docs.microsoft.com/en-us/azure/architecture/best-practices/background-jobs) | Utilize "Spot VMs" for compute | [Demand Shaping](./sustainable-engineering-principles.md#demand-shaping) | [How to use Spot VMs](https://docs.microsoft.com/en-us/azure/virtual-machines/spot-vms) | |
+| | For services with varied utilization patterns | Configure Autoscaling | [Energy Proportionality](./sustainable-engineering-principles.md#energy-proportionality) | [Autoscaling Documentation](https://docs.microsoft.com/en-us/azure/architecture/best-practices/auto-scaling) | |
+| | | Use serverless functions | [Energy Proportionality](./sustainable-engineering-principles.md#energy-proportionality) | [Serverless Architecture Design](https://docs.microsoft.com/en-us/azure/architecture/serverless-quest/serverless-overview) | |
+| | For services with co-located users (EG internal employee apps) | Select a datacenter region that is physicaly close to them | [Networking](./sustainable-engineering-principles.md#networking) | [Azure products available by region](https://azure.microsoft.com/en-us/global-infrastructure/services/) | |
+| | | Consider running an edge device | [Networking](./sustainable-engineering-principles.md#networking) | [Azure Stack Edge](https://azure.microsoft.com/en-us/products/azure-stack/edge/) | [Understanding edge tradeoffs](./sustainable-action-disclaimers.md#action-consider-running-an-edge-device) |
+| | For services with global users | Cache data close to them using a [CDN](https://en.wikipedia.org/wiki/Content_delivery_network) | [Networking](./sustainable-engineering-principles.md#networking) | [Benefits of a CDN](https://docs.microsoft.com/en-us/azure/frontdoor/front-door-overview#key-benefits) | |
+| | For systems sending data over the network | Send only the data that will be used | [Networking](./sustainable-engineering-principles.md#networking) | | |
+| | | Compress data to reduce the size | [Networking](./sustainable-engineering-principles.md#networking) | [Compressing and extracting files in .NET](https://docs.microsoft.com/en-us/dotnet/standard/io/how-to-compress-and-extract-files) | |
```

Comment:
Duplicate recommendation, but for different scenarios.  Someone using this checklist may find themselves sending data over the network but not using much storage.  I would not want them to miss the relevant recommendation simply because it happens to be mentioned in another scenario.